music21/romanText/tsvConverter.py

```diff
@@ -560,30 +607,43 @@ def toM21Stream(self):
         '''
         if not self.chordList:
             self.tsvToChords()
-        self.prepStream()
 
-        s = self.preparedStream
+        s = self.prepStream()
         p = s.parts.first()  # Just to get to the part, not that there are several.
 
+        if p is None:
+            # in case stream has no parts
+            return s
+
         for thisChord in self.chordList:
             offsetInMeasure = thisChord.beat - 1  # beats always measured in quarter notes
             measureNumber = thisChord.measure
             m21Measure = p.measure(measureNumber)
-
-            if thisChord.representationType == 'DCML':
-                thisChord._changeRepresentation()
+            if m21Measure is None:
+                # TODO: m21Measure should never be None if prepStream is
+                #   correctly implemented. We need to handle None to satisfy
+                #   mypy. If it *is* None, then there is a bug in the
+                #   implementation. What is correct behavior in this instance?
+                #   Raise a bug?
+                raise ValueError
```

Comment:
do not raise a class -- raise an instance with a message:  `raise ValueError('m21Measure should not be None')`  All code paths should be tested.